q/gradle-quickstart.sh

```diff
@@ -232,6 +223,13 @@ else
   exit 1
 fi
 
+# create gradle dir with gradle wrapper
+mkdir -p gradle/wrapper
+cat > gradle/wrapper/gradle-wrapper.properties <<EOF
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.3.3-all.zip
+EOF
```

Comment:
this is not really useful unless we also provide `./gradlew` scripts for actually running the wrapper

Reply:
I have added the instructions here: https://github.com/apache/flink/pull/18609/files#diff-b8bb4e796f587464ed5b9e215b5eade045eb10f087df8dfaea260d930aeb684fR415 for how one could use the wrapper.

Reply:
that implies that you already have gradle binaries available locally (so you can use them to generate the wrapper), which for most users won't hold true

Reply:
True, otherwise we need to provide the wrapper already in the project generated by the script.

Reply:
Or commit the wrapper in the sample project, but I think that's not nice....

Reply:
I think it doesn't harm to have the `gradle-wrapper.properties` there, and give an easy possibility to the user to download and use the wrapper. @dmvk what do you think?

Reply:
IMO it does neither harm or help. Personally I'd simply commit the wrapper in the project, all other projects do the same and it doesn't really add any overhead.